packages/appium/lib/config.js

```diff
@@ -157,8 +158,9 @@ async function showConfig () {
 }
 
 function getNonDefaultArgs (parser, args) {
-  return parser.rawArgs.reduce((acc, [, {dest, default: defaultValue}]) => {
-    if (args[dest] && args[dest] !== defaultValue) {
+  const defaultsFromSchema = getDefaultsFromSchema({prop: 'server', assignDefaults: true});
+  return parser.rawArgs.reduce((acc, [, {dest}]) => {
+    if (args[dest] && args[dest] !== defaultsFromSchema[dest]) {
```

Comment:
should we also check if `defaultsFromSchema[dest]` is defined?

Reply:
mm, probably